lib/recurly/relay.js

```diff
@@ -28,8 +28,17 @@ function relay (done) {
   self.origin = function () {
     var parser = document.createElement('a');
     parser.href = self.config.api;
-
-    return parser.origin;
+    
+    if (parser.origin) {
+      return parser.origin;
+    }
+    
+    // IE11 seems to append :443 to parser.host even when it's not there
+    if (parser.protocol === 'https:' && parser.port == 443) {
+      return parser.protocol+'//'+parser.hostname;
+    }
+    
+    return parser.protocol+'//'+parser.host;
```

Comment:
`host` would include the port, so `localhost:3000`, which is what you would normally want, but for some reason, IE11 includes `:443` if the protocol is `https`. So 

``` javascript
var parser = document.createElement('a');
parser.href = 'https://api.recurly.com/v1/';
console.log(parser.host); // api.recurly.com:443
```

This wouldn't work for the comparison, however I'm about to push another PR that will use the same function on both urls, which should make it more robust.